arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -0,0 +1,735 @@
+          "description": "The resource type."
+        },
+        "tagname": {
+          "type": "string",
+          "description": "The tag name."
+        },
+        "tagvalue": {
+          "type": "string",
+          "description": "The tag value."
+        }
+      },
+      "description": "Resource filter."
+    },
+    "ResourceGroupFilter": {
+      "properties": {
+        "tagName": {
+          "type": "string",
+          "description": "The tag name."
+        },
+        "tagValue": {
+          "type": "string",
+          "description": "The tag value."
+        }
+      },
+      "description": "Resource group filter."
+    },
+    "ManagementLockOwner": {
+      "properties": {
+        "applicationId": {
+          "type": "string",
+          "description": "The application Id of the management lock owner."
+        }
+      }
+    },
+    "ManagementLockProperties": {
+      "properties": {
+        "level": {
+          "type": "string",
+          "description": "The lock level of the management lock.",
+          "enum": [
+            "NotSpecified",
+            "CanNotDelete",
+            "ReadOnly"
+          ],
+          "x-ms-enum": {
+            "name": "LockLevel",
+            "modelAsString": true
+          }
+        },
+        "notes": {
+          "type": "string",
+          "description": "The notes of the management lock."
+        },
+        "owners": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementLockOwner"
+          },
+          "description": "The owners of the management lock."
+        }
+      },
+      "required": [ "level" ],
+      "description": "The management lock properties."
+    },
+    "ManagementLockObject": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagementLockProperties",
+          "description": "The properties of the lock."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The Id of the lock."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the lock."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the lock."
+        }
+      },
+      "required": [ "properties" ],
+      "description": "Management lock information.",
+      "x-ms-azure-resource": true
+    },
+    "ManagementLockListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementLockObject"
+          },
+          "description": "The list of locks."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "List of management locks."
+    },
+    "Resource": {
```

Comment:
Looks like Resource and SubResource model definitions are not being referenced anywhere in this spec. Please remove them.